clusters/l3-sqnc/capacity/magenta-node/values.yaml

```diff
@@ -0,0 +1,21 @@
+node:
+  chain: local
+  role: validator
+  dataVolumeSize: 100Gi
+  perNodeServices:
+    setPublicAddressToExternal:
+      enabled: false
+  keys:
+    - type: "babe"
+      scheme: "sr25519"
+      seed: babe_seed
```

Comment:
```suggestion
```

Reply:
because of this: `The seed ones are important as we are supplying those via the release.yaml` does it mean I should change the release.yaml to e.g. 
`    - kind: Secret
      name: babe-magenta-key
      valuesKey: babe_seed
      targetPath: node.keys[0].seed`
      
and then in `values.yaml` use the name instead of the `valuesKey`? 
`seed: babe-magenta-key`

Reply:
No, the secrets are already in place and correctly named.

Look at `clusters/l3-sqnc/secrets/magenta_capacity_node-keys.yaml`

Reply:
so I completely delete the `keys` section in values.yaml? Why? 
I thought the `seed` bit has a [purpose](https://github.com/digicatapult/helm-charts/blob/8e64f733e5bd78431f6833dd73af35d0c9e24ccf/charts/sqnc-node/values.yaml#L88C1-L91C29)

Reply:
Yes it does, but we are overriding it by supplying it from a secret in`release.yaml`

Reply:
ok so release.yaml overrides values.yaml and because we have keys defined there (release.yaml) already no need to specify again in values.yaml so we remove the whole section.

Reply:
Not the whole keys section, just the bit we are overriding, which is the seed.

Reply:
ok ... twas a bit painful but we got there